tictactoe/tictactoe-algorithm.md

```diff
@@ -0,0 +1,6 @@
+# tictactoe algorithm
+	1.Initialize the 3x3 array using the list.
+	2.Introduce first player is X and second player is O and ask user to select.
+	3.Check player X or O winning move else check opposite winning move and block placing X or O.
```

Comment:
Every time when user place peg check column wise, row and diagonal for any matching three peg for a particular user.